### ISSUE TITLE
change surround does not recognize double quotes "

### ISSUE THREAD

**Issue by Konfekt** [machakann/vim-sandwich#41]
With 

```vim
let g:sandwich#recipes = [
                \   {
                \     'buns':         ['"', '"'],
                \     'quoteescape':  1,
                \     'expand_range': 0,
                \     'nesting':      0,
                \     'match_syntax': 1,
                \     'input':        ['"', 'Q'],
                \   },
]
nmap     cs          <Plug>(operator-sandwich-replace)<Plug>(operator-sandwich-release-count)<Plug>(textobj-sandwich-query-a)
```
hitting `cs"` or `csQ` has no effect. The corresponding setup for single quotes work without any problems. I disabled targets.vim and removed the vim-sandwich setup, but still the error remains.

**Comment by machakann** [comment]
Hi! Do you reproduce it with the current version? Because, excuse me, I made a similar bug with commits in 3/14, and it is fixed just a few days ago (#40).

**Comment by Konfekt** [comment]
Thanks! Fixed by the single latest commit.
